src/PerfView/UserCommands.cs

```diff
@@ -230,6 +230,27 @@ public void SaveCPUStacks(string etlFileName, string processName = null)
             }
         }
 
+        /// <summary>
+        /// Save the entire CPU stacks from 'etlFileName' as a csv.  If the /process qualifier is present use it to narrow what
+        /// is put into the file to a single process.
+        /// </summary>
+        public void SaveCPUStacksAsCsv(string etlFileName, string processName = null)
```

Comment:
Might be worth adding an Async version of this method making use of `File.WriteAllTextAsync` under the hood to prevent blocking on IO.

Reply:
Good shout.  Should this be a separate method from the current one or be the default?

Reply:
I would add two different methods like System.IO's ``File.Read`` / ``File.Write`` methods but will leave it up to you. :)

Reply:
It doesn't look like the current version of dotnet PerfView uses supports File.WriteAllTextAsync?  Is this correct?  If not what would I be missing to enable this feature?  None of the other File.Write...Async are available either so this may be an issue on my end but I'm not sure.

Reply:
Sorry, you are right. Looks like the version of .NET PerfView is using is not high enough :(. Thank you very much for checking and all your effort.

Reply:
My pleasure!  Anything to make a tool better and faster :D